core/src/main/scala/org/apache/spark/status/api/v1/AllStagesResource.scala

```diff
@@ -142,7 +142,7 @@ private[v1] object AllStagesResource {
       index = uiData.taskInfo.index,
       attempt = uiData.taskInfo.attemptNumber,
       launchTime = new Date(uiData.taskInfo.launchTime),
-      duration = uiData.taskDuration,
+      duration = uiData.taskDuration(),
```

Comment:
this is unrelated and unnecessary

Reply:
Since i changed `taskDuration` below,if do not add `()` a compiler error will occur.

Reply:
odd, I thought for sure it'd be fine, then this LGTM

Reply:
Thanks @ajbozarth

Reply:
Here what if we call the REST API on history server to get stage info? Looks like we may still have this issue since we don't have last update time here, what do you think @ajbozarth ?

Reply:
You are right, @jerryshao .IIUC, the `ui` in `AllStagesResource.scala` is passed from `ApiRootResource` which also create `sparkUI` by `FSHistoryProvider`.So we can also get `lastUpdateTime` from this `ui` in `AllStagesResource` and pass to the `taskDuration` interface.I think it is another problem for REST?Should we fix here?

Reply:
Yes, if it is not a big change I think it should be fixed here. Because currently with this fix UI and REST API are inconsistent.